components/builder-api/src/server/handlers.rs

```diff
@@ -750,13 +750,25 @@ pub fn get_project_integration(req: &mut Request) -> IronResult<Response> {
     request.set_integration(opi);
 
     match route_message::<OriginProjectIntegrationGet, OriginProjectIntegration>(req, &request) {
-        Ok(integration) => Ok(render_json(status::Ok, &integration.get_body())),
+        Ok(integration) => {
+            let v: serde_json::Value = match serde_json::from_str(&integration.get_body()) {
+                Ok(v) => v,
+                Err(e) => {
+                    debug!("Error parsing to JSON. e = {:?}", e);
+                    return Ok(Response::with((status::UnprocessableEntity, "api-gpi:1")));
```

Comment:
To maintain consistency with the rest of our error codes, this should be `api:gpi:1`